### ISSUE TITLE
CBMC incorrectly provides false(unreach-call) in memleaks_test3_false-valid-free.i

### ISSUE THREAD

**Issue by lucasccordeiro** [marek-trtik/cbmc#12]
If we run CBMC as:

$cbmc memleaks_test3_false-valid-free.i --stop-on-fail

It correctly identifies the bug:

````
Violated property:
  file memleaks_test3_false-valid-free.i line 1440 function entry_point
  free argument must be dynamic object
  (void *)p == NULL || DYNAMIC_OBJECT((void *)p)


VERIFICATION FAILED
````

However, our script provides:

````
false(unreach-call)
````

It suggested that there is some bug in our script that parses the CBMC output.

**Comment by marek-trtik** [comment]
[xtest.zip](https://github.com/marek-trtik/cbmc/files/1470235/xtest.zip)

**Comment by lucasccordeiro** [comment]
This has been fixed via commit 551280c3bf5bb325b51e08c633d37430964cf240.

Now, we correctly report `FALSE(valid-free)` as follows:

Violated property:
  file /Users/lucasccordeiro/sv-benchmarks/c/ldv-memsafety/memleaks_test3_false-valid-free.i line 1440 function entry_point
  free argument must be dynamic object
  (void *)p == NULL || DYNAMIC_OBJECT((void *)p)


VERIFICATION FAILED
EC=10
FALSE(valid-free)
